src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -44,6 +45,9 @@ class BaseExpensiTextInput extends Component {
         if (this.props.autoFocus && this.input) {
             this.input.focus();
         }
+        if (this.input && this.props.name) {
```

Comment:
If there is no input we should return early instead of checking on like 48 and 45

Reply:
Yes, absolutely.